docs/source/en/generation_strategies.md

```diff
@@ -57,9 +57,10 @@ When you load a model explicitly, you can inspect the generation configuration t
 >>> model = AutoModelForCausalLM.from_pretrained("distilbert/distilgpt2")
 >>> model.generation_config
 GenerationConfig {
-    "bos_token_id": 50256,
-    "eos_token_id": 50256,
+  "bos_token_id": 50256,
+  "eos_token_id": 50256
 }
+<BLANKLINE>
```

Comment:
This is the way to pass the docstring.

See https://wiki.python.org/moin/MultiLineStringsInDocTest